text-serializer-gson/src/main/java/net/kyori/adventure/text/serializer/gson/ComponentSerializerImpl.java

```diff
@@ -135,6 +138,8 @@ private ComponentSerializerImpl(final Gson gson) {
         text = readString(in);
       } else if (fieldName.equals(TRANSLATE)) {
         translate = in.nextString();
+      } else if (fieldName.equals(TRANSLATE_FALLBACK)) {
```

Comment:
side note for any mergers: might be worth waiting until the json-serializer merge before merging this)

Reply:
I think we're going to merge this one first just to avoid rebasing external things unnecessarily